lib/email_notice.rb

```diff
@@ -0,0 +1,36 @@
+require "dotenv"
+require "pony"
+require_relative './database_connect'
+
+class EmailNotice
+
+  extend DatabaseConnect
+
+  def self.check_peep(peep, current_user_name)
```

Comment:
It's not really clear what you're checking here. I had to read this method a couple of times to work out what it's doing; I'd rather know from the method name.